packages/docs-nextra/components/doenet.tsx

```diff
@@ -1,6 +1,27 @@
 import React from "react";
-import dynamic from "next/dynamic";
+import {
+    DoenetViewer as DoenetViewerOrig,
+    DoenetEditor as DoenetEditorOrig,
+} from "@doenet/doenetml-iframe";
+import "@doenet/virtual-keyboard/style.css";
 
-export const DoenetML = dynamic(() => import("./doenetInternal"), {
-    ssr: false,
-});
+/**
+ * Render DoenetML in an iframe so that its styling/state is completely isolated from the page.
+ */
+export function DoenetViewer({
+    source,
+}: React.PropsWithChildren<{ source: string }>) {
+    return <DoenetViewerOrig doenetML={source} />;
+}
+
+export function DoenetEditor({
+    source,
+    showFormatter = false,
+}: React.PropsWithChildren<{ source: string; showFormatter?: boolean }>) {
+    console.log("source for editor", JSON.stringify(source));
+    return (
+        <div className="doenet-editor-container">
+            <DoenetEditorOrig doenetML={source} showFormatter={showFormatter} />
```

Comment:
I think the default view should be editor on top and rendered below, rather than side-by-side

Reply:
I'll check with Virginia, as I thought she wanted side-by-side. At least side-by-side does switch to top-to-bottom on mobile devices/narrow screens. We can make different versions or parametrized versions to allow the author to specify.

Reply:
I ended up making a `doenet-editor` that is vertical and a `doenet-editor-horiz` that is side-by-side.